Example/ios/Example.xcodeproj/project.pbxproj

```diff
@@ -562,7 +562,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = i386;
```

Comment:
This file probably should not be commited.

Reply:
It came with the RN 0.70 upgrade, TBH I don't understand why it excludes architectures in this way, seems to be a legacy thing since before ARM was properly supported in simulator. I can revert this, but pod install will add it back (or remove it if you're on Intel).